packages/waku/tests/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "..",
+    "outDir": "./dist",
+    "tsBuildInfoFile": "./dist/.tsbuildinfo",
```

Comment:
we are using the default for now.
```suggestion
```

Reply:
ok, fine

Reply:
I hope it emit to `dist`, so people can remove dist with it

Reply:
Yeah I understand that that sense, but `dist` also is something to publish, so it doesn't feel very appropriate in another sense.